### PR TITLE
Add option to hide label of choice questions

### LINKED ISSUE #124: Allow hiding the label of ChoiceQuestions and MultipleChoiceQuestions
Often, typically at the end of a form, there are one or more questions that ask the user to confirm something (terms of service, privacy policy, ...). Those questions usually consist of a single checkbox and a label, e.g.:

- [ ] I have read and understood the [terms of service]()

The current `MultipleChoiceQuestion` doesn't support this use-case very well: It requires a global question label which will be displayed on top.

The easiest way to bring this into the caluma form would be a "Hide label" option for Choice and MultipleChoiceQuestions. Then it is possible to add add a "Confirm" type of question as a MultipleChoiceQuestion with a single option.

### PR DESCRIPTION
**Pull request by Yelinz**
Add new meta option for Choice question to hide their labels.
Closes #124

### CODE REVIEW

File: addon/components/cfb-form-editor/question.js [hunk 1]
```diff
@@ -143,6 +143,9 @@ export default Component.extend(ComponentQueryManager, {
     if (model && model.subForm) {
       model.subForm = model.subForm.slug;
     }
+    if (model && model.meta && JSON.parse(model.meta).hideLabel) {
```
[the remark this answers is not in the document]
Comment (Yelinz) [reply addon/components/cfb-form-editor/question.js]: Still get a JSON string in the newest version.

[the remark this answers is not in the document]
Comment (fkm) [reply addon/components/cfb-form-editor/question.js]: Ok. Weird. But ok. As long as it works :-)

[the remark this answers is not in the document]
Comment (Yelinz) [reply addon/components/cfb-form-editor/question.js]: Changed it so it parses the meta when it querys, so we don't need the parse it later.
Fixes #139

File: tests/integration/components/cf-field-test.js [hunk 2]
```diff
@@ -78,4 +78,61 @@ module("Integration | Component | cf-field", function(hooks) {
...
+                  }
+                }
+              }
+            ]
+          }
+        }
+      }
+    });
+
+    this.set("question", document.fields[0]);
```
Comment (czosel) [inline tests/integration/components/cf-field-test.js]: Nitpick: This is a `field`, not a `question`. Fields contain a reference to a question and an answer, so calling the variable `question` is not ideal.

File: addon/components/cfb-form-editor/question.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (czosel) [reply addon/components/cfb-form-editor/question.js]: Nope, `meta` should be an object. @Yelinz try running `docker-compose pull caluma`

**Comment by czosel** [comment]
Another nitpick: I would mark the form field "Hide label" as required, because the toggle always has a value anyway.
Screenshot from 2019-04-18 16-56-37

File: addon/components/cfb-form-editor/question.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (Yelinz) [reply addon/components/cfb-form-editor/question.js]: I've tested it out with the newest version of the Caluma container and `meta` is now inconsistent, it sometimes is a Object and other times still a JSON string.

[the remark this answers is not in the document]
Comment (Yelinz) [reply addon/components/cfb-form-editor/question.js]: Actually, I was still on mirage.

[the remark this answers is not in the document]
Comment (Yelinz) [reply addon/components/cfb-form-editor/question.js]: Yes, it works Caluma delivers a Object, but there will be a problem when we use Mirage. Should I try to figure it out or leave it be?

[the remark this answers is not in the document]
Comment (fkm) [reply addon/components/cfb-form-editor/question.js]: Does it work with the real backend?

[the remark this answers is not in the document]
Comment (Yelinz) [reply addon/components/cfb-form-editor/question.js]: Yes

[the remark this answers is not in the document]
Comment (fkm) [reply addon/components/cfb-form-editor/question.js]: In that case I'd say leave it be and create a ticket so we can get back to it when we have more time.

File: tests/integration/components/cf-field-test.js (hunk 2 above)
Comment (Yelinz) [reply tests/integration/components/cf-field-test.js]: Changed the name

**Pull request merged**